test/crawler_test.py

```diff
@@ -0,0 +1,33 @@
+import requests
+import time
+
+def crawler_test(uid, upwd):
+    auth = (uid, upwd)
+    link = 'http://localhost:8000/manager/crawlers/'
+    data = {'year':'2018', 'semester':'1학기'}
+
+    # I don't know the reason, but it works only if requests is imported here.
+    # Do you know why?
+    import requests
+    print('Creating crawler', data, '...')
+    res = requests.post(link, auth=auth, data=data)
+    if res.status_code == 201:
+        print('Successfully created Crawler instance.')
```

Comment:
Is it okay to check the status code only? shouldn't we check that the crawler is crawling lectures of given year and semester?